dnf-behave-tests/features/module-platform.feature

```diff
@@ -42,6 +41,7 @@ Scenario: I can't list info for the pseudo-module
   And stdout matches line by line
    """
    ?Last metadata
+   ^dnf-ci-pseudo-platform-modular\s+
```

Comment:
A bit unrelated, but the `makecache` command changes the behaviour with regards to modules? How?

Also, is it not possible to simply use `And stdout is ...`? I believe the "?Last metadata" line is not needed. Unless  the `\s+` is actually a variable string, `And stdout is ...` is simpler and more effective?

Reply:
It is not related to modularity per se. dnf-ci-pseudo-platform-modular is just the name of repository being downloaded. Using "std out is" step is not possible because this step do not accept regexp and the output of the command is not exactly the same for each run. It looks like this:

dnf-ci-pseudo-platform-modular                  1.6 MB/s | 1.7 kB     00:00    
Unable to resolve argument pseudoplatform

I removed the Last metadata part though.

Reply:
Ok, merging.

I'm wondering why it sometimes downloads and sometimes doesn't though.